core/kazoo_attachments/src/kz_att_s3.erl

```diff
@@ -63,26 +64,36 @@ convert_kv({<<"etag">> = K, V}) ->
     {K, binary:replace(V, <<$">>, <<>>, ['global'])};
 convert_kv(KV) -> KV.
 
--spec get_map_values(kz_data:connection()) -> {ne_binary(), ne_binary(), ne_binary(), api_binary(), ne_binary()}.
+-spec get_map_values(kz_data:connection()) ->
+                            {ne_binary(), ne_binary(), ne_binary(), ne_binary(), api_binary()}.
 get_map_values(#{'bucket' := Bucket
                 ,'key' := Key
                 ,'secret' := Secret
                 }=Map) ->
     Path = maps:get('path', Map, 'undefined'),
     Host = maps:get('host', Map,  ?AMAZON_S3_HOST),
-    {Bucket, Key, Secret, Path, Host}.
+    {Key, Secret, Host, Bucket, Path}.
 
 -spec get_file_path(api_binary(), ne_binary(), ne_binary(), ne_binary()) -> list().
 get_file_path('undefined', DbName, DocId, AName) ->
     kz_util:to_list(list_to_binary([DbName, "/", DocId, "_", AName]));
 get_file_path(Path, DbName, DocId, AName) ->
     kz_util:to_list(list_to_binary([Path, "/", DbName, "/", DocId, "_", AName])).
 
--spec get_s3_values(ne_binary()) -> {ne_binary(), ne_binary(), ne_binary(), ne_binary(), api_binary()}.
-get_s3_values(S3) ->
+-spec get_s3_values(ne_binary(), kz_data:connection()) ->
+                           {ne_binary(), ne_binary(), ne_binary(), ne_binary(), api_binary()}.
+get_s3_values(S3, 'undefined') ->
     case binary_to_term(base64:decode(S3)) of
         {Key, Secret, Bucket, Path} ->
```

Comment:
@lazedo reverted back to same values, sorry was trying to aim for same output on both funs :)